modules/psap-special-resource-operator-installing-using-cli.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="installing-the-special-resource-operator-using-cli_{context}"]
+= Installing the Special Resource Operator by using the CLI
+
+As a cluster administrator, you can install the Special Resource Operator (SRO) by using the OpenShift CLI.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You installed the Node Feature Discovery (NFD) Operator.
+
+.Procedure
+
+. Create a namespace for the Special Resource Operator:
+
+.. Create the `Namespace` custom resource (CR) that defines the `openshift-special-resource-operator` namespace, and then save the YAML in the `sro-namespace.yaml` file:
```

Comment:
Create the -> Create a?

Reply:
Will change to "Create the following 'Namespace` custom resource... as was done in NFD.